### ISSUE TITLE
Hard code for luajit 2.1 binary name.

### ISSUE THREAD

**Issue by xpol** [moteus/lua-travis-example#15]
https://github.com/moteus/lua-travis-example/blob/master/.travis/setup_lua.sh#L56-L57

```bash
    ln -s $LUA_HOME_DIR/bin/luajit-2.1.0-alpha $HOME/.lua/luajit
    ln -s $LUA_HOME_DIR/bin/luajit-2.1.0-alpha $HOME/.lua/lua;
```
Should not hard coded as: `luajit-2.1.0-alpha`.

**Comment by ignacio** [comment]
Currently, I have the same problem in appveyor integration. Mike Pall said that he was going to do a final release soon, so whatever is fixed, will be broken soon.

**Comment by xpol** [comment]
#17 has tested here:
https://travis-ci.org/xpol/lua-rapidjson/jobs/79148512#L191

**Comment by ignacio** [comment]
Just saw your PR. Indeed, that's a better fix :) Please, forget about #16
